### PR TITLE
Feature/creating a data model

### PR DESCRIPTION
**Pull request by AbdelazizSoliman**
In this pull request I did the following:

- [ ] Created User, Post, Like, and Comment models.
- [ ] Created User, Post, Like, and Comment tables.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @AbdelazizSoliman 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have used the GitHub flow by submitting a pull request from a _feature_ branch to the _main_ branch as the **base** branch whereas you are required to use Gitflow which is from a _feature_ branch to the _development_ branch as the **base** branch. Kindly correct this. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AbdelazizSoliman** [comment]
Thank you @Gambit142, I updated the default branch to be dev branch.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,144 @@
...
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Abdelaziz Soliman**
+
+- GitHub: [@abdelazizsoliman](https://github.com/abdelazizsoliman)
+- Twitter: [@Abdelaz21958317](https://twitter.com/Abdelaz21958317)
+- LinkedIn: [Abdelaziz Soliman](https://www.linkedin.com/in/abdelaziz-soliman)
+
```
Comment (OyePriscilla) [inline README.md]: - [ ] Kindly fix the linkedln broken page to make your README file to be more engaging and professional. I got the error message below 👇 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @AbdelazizSoliman 👏 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Installed Ruby on Rails framework ✔️ 
Used database migration files to maintain database schema ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AbdelazizSoliman) [reply README.md]: Done, thank you.

Comment (rloterh) [reply README.md]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Soliman,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations! 🎉💯🌟 

## Highlights 🎯 
🎲 Good job on for introducing an improvement to the ReadMe links!
🎲 ReadMe looks great!

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbdelazizSoliman** [comment]
@rloterh,Thank you so much.

**Pull request merged**